### ISSUE TITLE
Import/Export Migration Tool error

### ISSUE THREAD

**Issue by kd8ssq** [webprofusion/certify#574]
I understand this is an experimental feature, just wanted you to be aware of the issue if you're not already.

When migrating data to a new server (Certify versions exact), I run the import tool and I receive the error 
"•Failed to write certificate to destination: C:\ProgramData\Certify\certes\assets\pfx\*******.pfx [Could not find a part of the path 'C:\ProgramData\Certify\certes\assets\pfx\*******.pfx'.] 🛑"

Not quite sure what's missing and why it's failing.  The rest of the cert information does seem to set up properly.  One other recommendation for the tool would be to refresh the cert list after an import so you don't have to close/re-open the app to see the newly imported certs.

**Comment by webprofusion-chrisc** [comment]
Thanks for testing! I wonder if that "*" was related to a wildcard certificate?

**Comment by kd8ssq** [comment]
Sorry Chris.  I put the * in the string above because I wasn't sure if it was some specialized cert value that needed to be private. The hashed value did show up in the output of the import, it just threw the error about writing the cert to the destination.  I did look in the Certify directory and noticed that the "certs" directory didn't exist.  I tried creating that to see if it would resolve the issue and it didn't change anything.

We're working on some server migrations and wanted to try this new tool out to see if it would work for moving our data from one server to the other.

Thanks
